core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPushdownFilterIntoWindow.java

```diff
@@ -61,6 +61,24 @@ public void testEliminateFilter()
                                 .maxRankingPerPartition(99)
                                 .partial(false),
                         values("a")));
+
+        ResolvedFunction rank = tester().getMetadata().resolveFunction(QualifiedName.of("rank"), fromTypes());
```

Comment:
If I may, could I request that instead of copying the blocks of test code, that we create parameterized functions such as:

    @Test
    public void testEliminateFilter()	{
      assertEliminateFilter("row_number");
      assertEliminateFilter("rank");
    }
    
    private void assertEliminateFilter(String rankingFunctionName) {
    
    }
    
  This is mainly because we will be adding dense rank at some point as well, and this will help us not to copy it 3x at that point.